### PR TITLE
Testing: fix test warnings

### PR DESCRIPTION
**Pull request by rdimaio**
Issue: #5522

Fixing test warning:
```
/opt/venv/lib64/python3.9/site-packages/_pytest/python.py:198: PytestReturnNotNoneWarning: Expected None, but tests/test_import_export.py::test_active returned True, which will be an error in a future version of pytest.  Did you mean to use `assert` instead of `return`?
```

### CODE REVIEW

**Comment by rdimaio** [comment]
Regarding https://github.com/rucio/rucio/pull/6450/commits/aaf52e578117b81f00faa920711c1f45c872260d - `test_active` fails the autocheck that runs on `sqlite`: https://github.com/rucio/rucio/actions/runs/7487468726/job/20380554506?pr=6450.

It's a bit unclear what the purpose of `test_active` is; @maany mentioned that this file was discussed in the past, will check to see what to do next

File: tests/test_import_export.py [hunk 1]
```diff
@@ -83,9 +83,7 @@ def reset_rses():
 
 def test_active():
```
Comment (maany) [inline tests/test_import_export.py]: My recommendation: remove the test altogether. It does not do anything useful. On the contrary, it tries to do things the testing framework was designed for i.e. selectively run/skip tests based on the environment.

Looking back at the commit where this  test was added, it looks more like a debug util for the developer working on the issue, that slipped through the cracks.
 
https://github.com/rucio/rucio/commit/e6bc1b3d6ba3c5f53ba85d425b12cf5ef4b28c1

**Review by maany — CHANGES_REQUESTED** [review]
Remove the test altogether

File: tests/test_import_export.py (hunk 1 above)
Comment (rdimaio) [reply tests/test_import_export.py]: thanks for checking :) I'll go ahead and take it out

**Review by maany — APPROVED** [review]
LGTM

**Pull request merged**